src/my-app.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference path="../bower_components/polymer-ts/polymer-ts.d.ts" />
```

Comment:
I couldn't figure out how to use plain ES imports. Didn't try that hard though...